neo/Helper.cs

```diff
@@ -271,5 +273,20 @@ internal static BigInteger WeightedAverage<T>(this IEnumerable<T> source, Func<T
                 yield return resultSelector(item, weight);
             }
         }
+
+        /// <summary>
+        /// load configuration with different Environment Variable
+        /// </summary>
+        /// <param name="config"></param>
+        /// <returns></returns>
+        public static IConfigurationRoot LoadConfig(string config)
+        {
+            var env = Environment.GetEnvironmentVariable("NEO_NETWORK");
+            var configFile = string.IsNullOrWhiteSpace(env) ? $"{config}.json" : $"{config}.{env}.json";
+            Plugin.Log(nameof(LoadConfig), LogLevel.Info, $"Config loading:{configFile}");
```

Comment:
I prefer to remove this line. Because this is a common method, this can cause the logs to become confusing. The caller should log by itself.

Reply:
Sure,I've removed it.